crates/libcontainer/src/container/tenant_builder.rs

```diff
@@ -32,6 +32,22 @@ const NAMESPACE_TYPES: &[&str] = &["ipc", "uts", "net", "pid", "mnt", "cgroup"];
 const TENANT_NOTIFY: &str = "tenant-notify-";
 const TENANT_TTY: &str = "tenant-tty-";
 
+fn get_path_from_spec(spec: &Spec) -> Option<String> {
+    let process = match spec.process() {
+        Some(p) => p,
+        None => return None,
+    };
+    let env = match process.env() {
+        Some(e) => e,
+        None => return None,
+    };
+    env.iter()
+        .find(|e| e.starts_with("PATH"))
+        .iter()
+        .next()
+        .map(|s| s.to_string())
```

Comment:
Should we SHOULD choose the first one if multiple PATH envs are in the config?

Reply:
As per the runtime spec, env vars should follow https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08_01 , which mentions that

>  If more than one string in an environment of a process has the same name, the consequences are undefined.

In such case instead of first maybe we should take the last occurrence of path? That usually follows the common intuition of last specified value taking preference.

Reply:
I agree with that. We should pick up the last one and leave the comment.

Reply:
fixed, now last value of path is taken. Please take a look